src/libraries/System.Net.WebSockets.Client/tests/System.Net.WebSockets.Client.Tests.csproj

```diff
@@ -71,4 +71,8 @@
   <ItemGroup>
     <PackageReference Include="System.Net.TestData" Version="$(SystemNetTestDataVersion)" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
```

Comment:
Is this needed? the project doesn't seem to be built for .NET Framework, unless I'm missing something 🤔
https://github.com/dotnet/runtime/blob/9ea5d2d6cdc536fbe1dacc477ef2b2dea2b4845c/src/libraries/System.Net.WebSockets.Client/tests/System.Net.WebSockets.Client.Tests.csproj#L4

Reply:
Removed condition

Reply:
It breaks builds, reverted